### DISCUSSION TITLE
Is TGI not available for ARM64 platform?

### DISCUSSION THREAD

**Discussion by SMAntony** [huggingface/text-generation-inference#2247] [Q&A]
I wanted to use TGI in a arm64 platform but couldnt. Are there any plans in their roadmap to add support for arm64?

**Comment by fawern** [comment]
As of now, Hugging Face's Text Generation Inference (TGI) primarily supports x86_64 architectures and does not officially support the ARM64 platform. There have been community inquiries regarding ARM64 support, but no official plans have been announced. 
GITHUB

Some users have attempted to build TGI for ARM64 by modifying Dockerfiles and build scripts, but these efforts are experimental and not officially supported. 
HUGGING FACE

If you require TGI on an ARM64 platform, you might consider exploring these community-driven approaches, keeping in mind that they may involve significant customization and potential stability issues.
